spec/support/features/session_helper.rb

```diff
@@ -50,6 +50,7 @@ def sign_in_user(user = create(:user))
     end
 
     def sign_in_before_2fa(user = create(:user))
+      allow(FeatureManagement).to receive(:prefill_otp_codes?).and_return(true)
```

Comment:
why are we adding this here? Don't see that we are removing a similar line in any of the specs, which leads me to believe it might not be necessary?